core-js/src/main/javascript/cdf/components/UnmanagedComponent.js

```diff
@@ -216,6 +216,20 @@ define([
     //priority: 5,
 
     /**
+     * @summary The queryDefinition Object.
+     * @description The queryDefinition Object.
+     *
+     * @type {object}
+     */
+     queryDefinition: undefined,
+    /**
+     * @summary The chartDefinition Object.
+     * @description The chartDefinition Object.
```

Comment:
This needs to be more descriptive. We need to add `link` (and maybe `see`) tag pointing to the queryDefinition property, because they represent the same kind of definitions `object`.